### PR TITLE
MODLOGIN-64

### PR DESCRIPTION
**Pull request by OleksiiMaksymov**
MODLOGIN-64 Add CRUD REST implementation to mod-event-config

* Add implementation `/eventConfig` which is accessible through OKAPI
* Add endpoint GET: /eventConfig  
* Update rest tests

### CODE REVIEW

**Comment by OleksiiKuzminov** [comment]
resolve conflict plz

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+ * Add endpoint GET: /eventConfig  
+ * Update rest tests
+ 
+  API: 
+  
+  | METHOD |  URL                          | DESCRIPTION                                                       |
+  |--------|-------------------------------|-------------------------------------------------------------------|
+  | POST   | /eventConfig                  | Create new event config in storage                                |
+  | GET    | /eventConfig                  | Get all event configs or accepted by query                        |
+  | GET    | /eventConfig/id               | Get event config from storage                                     |
```
Comment (kostyantyn-kh) [inline NEWS.md]: add curly braces for all ids - > {id}

File: ramls/event_config.raml [hunk 2]
```diff
@@ -12,6 +12,7 @@ documentation:
 types:
   errors: !include raml-util/schemas/errors.schema
   eventEntity: !include event_entity.json
+  eventEntries: !include event_entries.json
```
Comment (kostyantyn-kh) [inline ramls/event_config.raml]: rename event_entries to event_entity_collection to make it more clear how entity is connected with entity_collection

File: NEWS.md (hunk 1 above)
Comment (OleksiiMaksymov) [reply NEWS.md]: Fix

File: ramls/event_config.raml (hunk 2 above)
Comment (OleksiiMaksymov) [reply ramls/event_config.raml]: thanks for the comments!
Done

File: src/main/java/org/folio/rest/impl/InitAPIs.java [hunk 3]
```diff
@@ -3,10 +3,14 @@
 import io.vertx.core.*;
```
Comment (opochepam) [inline src/main/java/org/folio/rest/impl/InitAPIs.java]: please avoid wildcard imports

File: src/main/java/org/folio/rest/impl/EventConfigAPIs.java [hunk 4]
```diff
@@ -3,28 +3,44 @@
...
 
 import javax.ws.rs.core.Response;
+import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 
-import static org.folio.util.EventConfigUtils.createFutureResponse;
+import static javax.ws.rs.core.MediaType.TEXT_HTML;
+import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+import static org.folio.util.EventConfigUtils.*;
```
Comment (opochepam) [inline src/main/java/org/folio/rest/impl/EventConfigAPIs.java]: wildcard import

File: src/main/java/org/folio/rest/impl/InitAPIs.java (hunk 3 above)
Comment (OleksiiMaksymov) [reply src/main/java/org/folio/rest/impl/InitAPIs.java]: fix

File: src/main/java/org/folio/rest/impl/EventConfigAPIs.java (hunk 4 above)
Comment (OleksiiMaksymov) [reply src/main/java/org/folio/rest/impl/EventConfigAPIs.java]: Fix

**Comment by OleksiiMaksymov** [comment]
Done

**Pull request merged**